var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -3,14 +3,219 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import glob
 import os
 
-from llnl.util.filesystem import join_path
+from llnl.util.filesystem import find, join_path, working_dir
 
-from spack import *
+import spack.build_environment
+from spack.build_systems.makefile import MakefilePackage
+from spack.directives import *
+from spack.pkgkit import *
```

Comment:
I think this is something we've been meaning to add to all packages at some point, not sure if we ever made any progress on that

Reply:
Yeah, it's on my list.  That and getting things out of the module, and into package instances or similar so we can actually typecheck these darned things would make me really happy.  It keeps getting pushed back by functional stuff for build though.  At least putting it in as I go gets most of the type checking failures out for the ones I'm touching.

Reply:
If we could get it in before 0.18 (sometime in the next week) that would be amazing. It's hard to add after because it becomes difficult to backport.

Reply:
I can try to dig out my old rope script and just do it, it basically can't break anything, so I think it's mainly a question of getting it in and writing something to make sure every new package has it.  Maybe add an isort rule? Will look into it.